README.md

```diff
@@ -1,6 +1,7 @@
 # Visual Studio Code - Open Source ("Code - OSS")
 
 <!-- [![Build Status](https://dev.azure.com/vscode/VSCode/_apis/build/status/VS%20Code?branchName=master)](https://aka.ms/vscode-builds) -->
+
```

Comment:
I don't think we need a space here since the commented build status line is part of the existing group of badges